strawberry_django/subscriptions/utils.py

```diff
@@ -0,0 +1,11 @@
+def get_group(instance):
+    return f"{instance.__class__.__name__}"
```

Comment:
I don't think the class name is enough, as 2 models from different modules can have similar names.

You can get `instance.__class__._meta` and use `f"{meta.app_label}.{meta.model_name}"`

Also, maybe prefix it with `strawberry-subscription-{}` would be nice to make sure this doesn't conflict by mistake with anything else?

Reply:
Sounds like a plan.  I'll add that.